Gemfile.lock

```diff
@@ -5,26 +5,23 @@ GEM
       base64
       nkf
       rexml
-    activesupport (7.2.2.1)
```

Comment:
The Gemfile being changed seems a little displaced since this PR is not adding any new package

Reply:
🤔